include/dlaf/lapack/tile.h

```diff
@@ -396,20 +396,24 @@ void assertExtendInfo(F assertFunc, cusolverDnHandle_t handle, CusolverInfo<T>&&
 }
 
 template <class T>
-dlaf::BaseType<T> lange(cusolverDnHandle_t, const lapack::Norm norm, const Tile<T, Device::GPU>& a) {
+dlaf::BaseType<T> lange(cusolverDnHandle_t handle, const lapack::Norm norm,
+                        const Tile<T, Device::GPU>& a) {
   DLAF_STATIC_UNIMPLEMENTED(T);
+  dlaf::internal::silenceUnusedWarningFor(handle, norm, a);
```

Comment:
Do you still get the error if you remove the names of the variables? (e.g. handle, norm, a)

If no, I suggest we use that instead of introducing a function (`silenceUnusedWarningFor()`)